src/computer.c

```diff
@@ -123,11 +134,16 @@ enum OperationReturnCode ld_mod_computer(LDHandle *handle, const char *name, con
  */
 enum OperationReturnCode ld_rename_computer(LDHandle *handle, const char *old_name, const char *new_name, const char *parent)
 {
-    TALLOC_CTX *talloc_ctx = talloc_new(NULL);
+    TALLOC_CTX *talloc_ctx;
```

Comment:
Consider using TALLOC_CTX *talloc_ctx = NULL; instead of TALLOC_CTX *talloc_ctx;.